regions/shapes/tests/test_rectangle.py

```diff
@@ -25,6 +25,36 @@ def wcs():
     header = fits.getheader(filename)
     return WCS(header)
 
+def test_corners():
+
+    xc,yc = 2,2
+    angle = 30*u.deg
+    width = 2
+    height = 1
+    reg = RectanglePixelRegion(center=PixCoord(xc, yc),
+                               width=width, height=height, angle=angle)
+
+    y1 = yc + np.cos(angle) * height/2 + np.sin(angle) * width/2
+    x1 = xc + np.cos(angle) * width/2 - np.sin(angle) * height/2
+
+    assert (x1, y1) in reg.corners
+
+    reg = RectanglePixelRegion(center=PixCoord(xc, yc),
+                               width=width, height=height, angle=90*u.deg)
+    # simple case: rotate by 90
+    assert {(2.5, 3), (1.5, 3), (2.5, 1), (1.5,1)} == set(map(tuple,
```

Comment:
Why use a Python set? Just assert on the list or array that comes out, then you also check the order of corners, i.e. establish exactly what the method does.

Reply:
laziness.  Which is weird because it was a lot more text the wrong way.

What I really wanted to do was make a test that was... iso...something?  isomorphic? under rotation, e.g., such that:
[1,2,3,4] == [2,3,4,1] != [1,3,2,4]
since technically that's what matters for a polygon, but I never figured out the appropriate term and moved on...